features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -161,6 +165,10 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
         data: applicationInboundConfigs
     } = useGetApplicationInboundConfigs(applicationId, SupportedAuthProtocolName.OIDC, !!applicationId);
 
+    const {
+        mutate: mutateAuthenticatorConnectedApps
```

Comment:
Shall we consider using SWR's global mutate function or a custom hook to mutate the data object? Since we don't have a requirement to fetch the data here this could be an unnecessary overhead.